model/articles.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection.js");
+const data = require('../db/data/test-data')
+const fs = require('fs')
+const path = require('path')
+
+
```

Comment:
There is no need for three of these imports, they're not being used, they'll never be used and shouldn't be here.